pandas/tests/plotting/test_frame.py

```diff
@@ -3229,6 +3229,20 @@ def test_subplots_sharex_false(self):
         tm.assert_numpy_array_equal(axs[0].get_xticks(), expected_ax1)
         tm.assert_numpy_array_equal(axs[1].get_xticks(), expected_ax2)
 
+    def test_plot_no_rows(self):
+        df = pd.DataFrame({"foo": [1]}, index=pd.Index(data=[], dtype=int), dtype="int")
+        assert df.empty
+        ax = df.plot()
+        assert len(ax.get_lines()) == 1
+        line = ax.get_lines()[0]
+        assert len(line.get_xdata()) == 0
+        assert len(line.get_ydata()) == 0
+
```

Comment:
Can you add a test for Series.plot with a  length-zero Series? I assume it does the same thing?

Reply:
Done.